src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,23 @@
+package seedu.addressbook.ui;
+
+public class Formatter {
+    /** A decorative prefix added to the beginning of lines printed by AddressBook */
```

Comment:
Good to have simple documentation for the class. Note that you are supposed to place these on top of the class rather than inside.